deployment/ansible/roles/raster-foundry.docker/tasks/main.yml

```diff
@@ -6,6 +6,4 @@
   user: name="{{ item }}"
         groups=docker
         append=yes
-  with_items:
```

Comment:
Yeah, I could also see adding this to the base docker role and having it only run when `docker_users` is defined because this is such a common task.